Formula/php@7.1.rb

```diff
@@ -0,0 +1,384 @@
+class PhpAT71 < Formula
+  desc "General-purpose scripting language"
+  homepage "https://secure.php.net/"
+  url "https://php.net/get/php-7.1.15.tar.xz/from/this/mirror"
+  sha256 "0e17192fb43532e4ebaa190ecec9c7e59deea7dadb7dab67b19c2081a68bd817"
+
+  keg_only :versioned_formula
```

Comment:
We is this keg_only ? Where as it use to not be?

Reply:
For versioned formulae, we prefer to use `keg_only :versioned_formula` instead of `conflicts_with` since it lets people easily install more than one version at the same time. You can always `brew link --force php@7.1`, which is persistent across upgrades.